### PR TITLE
Add process module to cnex

### PR DESCRIPTION
**Pull request by gitlarryf**

### CODE REVIEW

File: exec/cnex/lib/process_win32.c [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    sa.nLength = sizeof(sa);
+    sa.bInheritHandle = TRUE;
+    sa.lpSecurityDescriptor = NULL;
+    if (!CreatePipe(&out_read, &out_write, &sa, 0)) {
+        error = GetLastError();
+        char szErr[32];
+        snprintf(szErr, 32, "%d", error);
+        exec->rtl_raise(exec, "ProcessException", szErr, BID_ZERO);
+        return;
+        //goto bail;
```
Comment (ghewgill) [inline exec/cnex/lib/process_win32.c]: Remove commented code.

File: exec/cnex/util.h [hunk 2]
```diff
@@ -23,4 +23,11 @@ typedef int BOOL;
 #define TRUE (!FALSE)
 #endif
 
+#ifndef max
+#define max(a, b) (((a) > (b)) ? (a) : (b))
+#endif
+#ifndef min
```
Comment (ghewgill) [inline exec/cnex/util.h]: Don't include a definition unless you actually need it (min).

File: exec/cnex/lib/process_win32.c (hunk 1 above)
Comment (gitlarryf) [reply exec/cnex/lib/process_win32.c]: Opps.  Done.

File: exec/cnex/util.h (hunk 2 above)
Comment (gitlarryf) [reply exec/cnex/util.h]: Removed.

**Pull request merged**